static/vendor/ext-searchbox.js

```diff
@@ -14,7 +14,7 @@ var SearchBox = function (editor, range, showReplaceForm) {
     dom.buildDom(["div", { class: "ace_search right" },
         ["span", { action: "hide", class: "ace_searchbtn_close" }],
         ["div", { class: "ace_search_form" },
-            ["input", { class: "ace_search_field", placeholder: "Search for", spellcheck: "false" }],
+            ["input", { class: "ace_search_field", placeholder: "Search...", spellcheck: "false" }],
```

Comment:
The way we have this now, it cannot be localized. Would be nice if that is also possible (will open a new issue for that)